webapp/locales/en.json

```diff
@@ -251,7 +254,12 @@
   "post": {
     "name": "Post",
     "moreInfo": {
-      "name": "More info"
+      "name": "More info",
+      "title": "More information",
+      "description": "Here you can find more information about this topic.",
+      "titleOfCategoriesSection": "Categories",
+      "titleOfHashtagsSection": "Hashtags",
+      "titleOfRelatedContributionsSection": "Related posts"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-13T10:33:58Z" title="Tuesday, August 13th 2019, 12:33:58 pm +02:00">Aug 13, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -251,7 +254,12 @@
   "post": {
     "name": "Post",
     "moreInfo": {
-      "name": "More info"
+      "name": "More info",
+      "title": "More information",
+      "description": "Here you can find more information about this topic.",
+      "titleOfCategoriesSection": "Categories",
+      "titleOfHashtagsSection": "Hashtags",
+      "titleOfRelatedContributionsSection": "Related Posts"
```


Let's respect our capitalization convention @Tirokk? 
we decided we'd capitalize the first word and use lowercase for all other words in a sentence, right?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-14T06:18:52Z" title="Wednesday, August 14th 2019, 8:18:52 am +02:00">Aug 14, 2019</time>_

---


Thank @mattwr18 !🤗
Done